docs/index.md

```diff
@@ -165,6 +166,11 @@ $ eval "$(docker-machine env dev)"
 $ docker ps
 ```
 
+> **Note**: If you are using `fish`, or a Windows shell such as
+> Powershell/`cmd.exe` the above method will not work as described.  Instead,
+> see [the `env` command's documentation](http://docs.docker.com/machine/#env)
```

Comment:
I updated the `http` to `https` but I'm a little hesistant to move over to using relative URL paths for now because I'm not 100% sure the machine docs will be on the main Docker site forever.